### ISSUE TITLE
Walk script doesn't change Tux's direction

### ISSUE THREAD

**Issue by Alzter** [SuperTux/supertux#1035]
![2019-01-08_16-23-59](https://user-images.githubusercontent.com/22513909/50813389-ed917880-1361-11e9-9c76-204a39795691.gif)
TL;DR: Tux can moonwalk.

**Comment by Alasdairbugs** [comment]
i think they should keep that as an easter egg, thats hilarious XD

**Comment by tobbi** [comment]
Can you attach the script in question here, please?

**Comment by Alzter** [comment]
The walking script commonly used is: `Tux.walk(float speed)`

Although personally I prefer the scripted controller scripts as they control Tux more naturally.

**Comment by Rusty-Box** [comment]
I think an easy way to fix this is to just add `Tux.set_dir(true);` (which for some reason just uses "left" and "right") to force Tux to turn right when touching the script trigger

**Comment by Paulo-Tux** [comment]
> 
2019-01-08_16-23-59
TL;DR: Tux can moonwalk
> Eu fiz um Backflip nessa zona e o Tux ficou paralisado em vez de acontecer esse bug. (I did a Backflip in that zone and Tux got stuck instead of this bug.)

**Comment by Paulo-Tux** [comment]
I did a Backflip in the Tower of Ghosts walk script and Tux instead of being paralyzed like in Nolok's Castle, this bug also happened.

**Comment by Paulo-Tux** [comment]
From what I've seen, the walk script just makes Tux go to the right and in the seq. trigger Tux can go left and right at will (even with backflips) and in my opinion, the Super Tux team should copyright the seq. trigger to solve the walk script problem. Watch the parts of my video for more details that I may not have mentioned.

https://user-images.githubusercontent.com/126248752/232251555-2e512a5a-d08b-442c-b2c0-063a1abeb7c4.mp4

**Comment by Rusty-Box** [comment]
I think this is a very useful thing for cutscenes, like having To jump backwards in shock/surprise without turning around. Closing this.
